### ISSUE TITLE
Custom baselayer is lost upon browser refresh

### ISSUE THREAD

**Issue by HarelM** [openstreetmap/iD#4852]
I totally dig the fact that you can set a custom base layer for iD to use using a URL.
Here's a link I'm using from my site to allow an OSM user to edit a feature:
https://www.openstreetmap.org/edit?node=278477259#background=custom:https://israelhiking.osm.org.il/Hebrew/Tiles/{z}/{x}/{y}.png

Unfortunately when surfing to the above page and then refreshing it, the custom base layer is lost...
This is probably due to the fact that the address is being changed after the page finishes loading.
Will it be possible to facilitate for keeping the custom layer after refresh as well?

**Comment by bhousel** [comment]
> This is probably due to the fact that the address is being changed after the page finishes loading.

Yes, this url-rewriting issue might be something we need to change on the openstreetmap-website side, but I agree that it's expected that a refresh would not change the user's imagery.
